hubblestack/extmods/fileserver/azurefs.py

```diff
@@ -201,9 +201,19 @@ def update():
         except Exception as exc:
             log.exception('Error occurred fetching blob list for azurefs')
 
-            if not __opts__['delete_inaccessible_azure_containers'] or not "<class 'azure.common.AzureHttpError'>" in str(type(exc)) :
+            if not __opts__['delete_inaccessible_azure_containers'] \
+               or ( not "<class 'azure.common.AzureHttpError'>" in str(type(exc)) \
+                    and \
+                    not "<class 'azure.common.AzureMissingResourceHttpError'>" in str(type(exc))
+                    ):
                 continue
-            if '<AuthenticationErrorDetail>Signature did not match.' in str(exc):
+
```

Comment:
'<AuthenticationErrorDetail>Signature did not match.' is not a good string. THere are different error strings for different scenarios. Specially when we use stored access policy with azurefs, there are many other error strings. These newly added 3 error codes should cover them all.